packages/shared/lib/contexts/governance/stores/is-governance-transferring.store.ts

```diff
@@ -0,0 +1,18 @@
+import { writable } from 'svelte/store'
+
+export const isGovernanceTransferring = writable<{ [accountId: number]: boolean }>({})
```

Comment:
Rename this file consistently to use something like `hasPendingGovernanceTransaction`
```suggestion
export const hasPendingGovernanceTransaction = writable<{ [accountId: number]: boolean }>({})
```